### ISSUE TITLE
Conda build fails with arcane error

### ISSUE THREAD

**Issue by pgrinaway** [choderalab/perses#387]
The conda build is [currently failing](https://travis-ci.org/choderalab/perses/jobs/270572699#L1304) with a somewhat arcane error:

```
Traceback (most recent call last):
  File "/home/travis/miniconda/bin/conda-build", line 4, in <module>
    import conda_build.cli.main_build
  File "/home/travis/miniconda/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 16, in <module>
    import conda_build.api as api
  File "/home/travis/miniconda/lib/python2.7/site-packages/conda_build/api.py", line 22, in <module>
    from conda_build.config import Config, get_or_merge_config, DEFAULT_PREFIX_LENGTH as _prefix_length
  File "/home/travis/miniconda/lib/python2.7/site-packages/conda_build/config.py", line 15, in <module>
    from .conda_interface import root_dir, root_writable, cc, subdir, platform
  File "/home/travis/miniconda/lib/python2.7/site-packages/conda_build/conda_interface.py", line 22, in <module>
    from conda.resolve import MatchSpec, NoPackagesFound, Resolve, Unsatisfiable, normalized_version  # NOQA
ImportError: cannot import name NoPackagesFound
```

Thoughts?

**Comment by jchodera** [comment]
Have you tried [unpinning `conda-build` here](https://github.com/choderalab/perses/blob/master/devtools/travis-ci/install.sh#L17)?

**Comment by jchodera** [comment]
Also, how about we drop Python 2.7 support and just go with 3.5 and 3.6?

**Comment by jchodera** [comment]
Looks like this is fixed by pinning `conda-build` to 2.1.17 in #386.

**Comment by pgrinaway** [comment]
Closing as resolved; reopen if not.
